src/app/core/admin/admin-list-manager/admin-list-manager.component.ts

```diff
@@ -36,41 +36,56 @@ import { MatSelectModule } from "@angular/material/select";
   styleUrl: "./admin-list-manager.component.scss",
 })
 export class AdminListManagerComponent implements OnInit {
-  @Input() items: (string | ColumnConfig)[] = [];
+  @Input() items: ColumnConfig[] = [];
   @Input() entityType: EntityConstructor;
   @Input() fieldLabel: string;
   @Input() templateType: "default" | "filter" = "default";
-  @Input() activeFields: (string | FormFieldConfig)[] = [];
+  @Input() activeFields: ColumnConfig[] = [];
+  @Input() onlyIDs: boolean = false;
+  @Output() itemsChange = new EventEmitter<ColumnConfig[]>();
+  @Output() idsChanges = new EventEmitter<string[]>();
```

Comment:
if we have an output for itemsChange and one for idsChange (which seems a smart appraoch, actually), then I think the `onlyIDs` input as a toggle is not needed, is it?

Reply:
We need something to differentiate between what to emit, for eg, the full config or just the IDs, so in that way, it’s helpful